workflow-job

```diff
@@ -538,7 +538,7 @@ flows["${sha1}"] = {
                 rabbitMQProductExists = fileExists 'dockerfiles/run/guice/cassandra-rabbitmq-ldap'
                 memoryProductExists = fileExists 'dockerfiles/run/guice/memory'
 
-                sh "docker volume create ${m2volume}"
+                sh "docker volume create --driver local --opt type=tmpfs --opt device=tmpfs --opt o=size=400m ${m2volume}"
```

Comment:
hey, you sneakily put it back :trollface:

Reply:
Arghh